src/main/java/uk/gov/cshr/controller/ResetController.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.cshr.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.cshr.service.ResetService;
+
+@Controller
+@RequestMapping("/reset")
+public class ResetController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(InviteController.class);
+
+    @Autowired
+    private ResetService resetService;
+
+    @GetMapping
+    public String reset() {
+        return "user-reset";
+    }
+
+    @PostMapping
+    public String reset(@RequestParam(value = "email") String email) throws Exception {
+        LOGGER.info("Resetting {} ", email);
```

Comment:
INFO - Where do we check email actually exists before kicking off the process?
We should log negative lookups as info also

Reply:
Yep we're not just right now. Will be in next PR (or can throw it in now if you'd like?)

Reply:
Complete the task as we go